test/JDBC/input/BABEL-4217-vu-verify.sql

```diff
@@ -0,0 +1,77 @@
+SELECT * FROM BABEL_4217_vu_prepare_t1;
```

Comment:
We need to add this test into `latest/schedule` and `master/schedule` files.

Reply:
Updated